apps/admin_page/src/pages/AiModels.tsx

```
@@ -126,12 +142,79 @@ const AiModels: React.FC = () => {
+            <button
+              onClick={() => setSearchQuery('')}
+              className="absolute right-5 top-1/2 -translate-y-1/2 p-1 hover:bg-brand-50 rounded-full text-brand-300 hover:text-brand-600 transition-colors"
+            >
+              <X size={18} />
+            </button>
+          )}
+        </div>
+      </div>
+
+      {(() => {
+        const { filteredConfigs, paginatedConfigs, totalPages } = useMemo(() => {
```

Biome (2.4.4)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

(lint/correctness/useHookAtTopLevel)